### ISSUE TITLE
Using PADProductConfiguration to overwrite trialText for better localisation

### ISSUE THREAD

**Issue by Iomegan** [PaddleHQ/Mac-Framework-V4#39]
Hi,

Currently` PADProductConfiguration` is only being used to for the first launch of the app. However it would be great if it could be used afterwards. I'd like to use it for setting a `trialText` that is localisable. 

Right now the _Trial Greeting_ from the product page is being used and I can only specify one language there. Usually English.

```
let defaultProductConfig = PADProductConfiguration()
defaultProductConfig.trialText = NSLocalizedString("Localisable trial text.", comment: "")
```

<img width="493" alt="bildschirmfoto 27" src="https://user-images.githubusercontent.com/1634465/49744259-ac437580-fc9c-11e8-8f9b-72b9ab38c1f7.png">

**Comment by GregularExpressions** [comment]
We have a solution for this, it'll be coming in an upcoming release.

**Comment by GregularExpressions** [comment]
Hey @Iomegan here's a [v4.0.9 Beta](https://s3.amazonaws.com/paddle-production-sdk-releases/macOS/v4/beta/macOS-sdk-beta.zip) that has an approach to solving this.

`PADProduct` objects now have a new property of `localizedTrialText` which will override the remotely supplied trial text when it is set.

Let us know if that is a suitable solution for you and if you spot any problems.

**Comment by Iomegan** [comment]
Thanks for the implementation.

It works if I set the `localizedTrialText` of the `PADProduct`. However it seems not to work via the `PADProductConfiguration`.

Here's my code:
```
lazy var paddleProduct: PADProduct = {
        let paddleProduct = PADProduct(productID: "490109", productType: PADProductType.sdkProduct, configuration: defaultProductConfig)!
        return paddleProduct
    }()
 
lazy var paddle: Paddle = {
        return Paddle.sharedInstance(withVendorID: "000", apiKey: "abc", productID: "000000", configuration: defaultProductConfig, delegate: self)!
    }()

lazy var defaultProductConfig: PADProductConfiguration = {
        let productConfig = PADProductConfiguration()
        productConfig.productName = "Network Radar"
        productConfig.vendorName = "Witt Software"
        productConfig.trialType = .timeLimited
        productConfig.trialLength = 30
        productConfig.localizedTrialText = "PADProductConfiguration test"
        return productConfig
    }()

...

func applicationDidFinishLaunching(_ notification: Notification) {
     paddleProduct.refresh({ (delta: [AnyHashable : Any]?, error: Error?) in
           self.paddle.showProductAccessDialog(with: self.paddleProduct)
       })
}
```

**Comment by mpaddle** [comment]
Hi @Iomegan, 

The product configuration would only be used when the product data has never been refreshed before (which may happen as part of Paddle initialisation). Could you remove your `<productID>.padl` and try again, please?

**Comment by Iomegan** [comment]
I can confirm that it works that way after removing the padl file. 

Cheers

**Comment by GregularExpressions** [comment]
Shipped in [v4.0.9](https://github.com/PaddleHQ/Mac-Framework-V4/releases/tag/v4.0.9) 👍
